README.md

```diff
@@ -8,6 +8,12 @@ framework. The goal is to help pick up on completely avoidable accessibility
 issues (such as alt text for images, labels in forms, and color contrast) during
 your testing pipeline.
 
+## Installation
+
+Within your Ember app, use the [Ember CLI](http://ember-cli.com/) to install this test suite as an add-on:
+
+`$ ember install ember-a11y-testing`
```

Comment:
Can you change this to use a [code fence](https://help.github.com/articles/creating-and-highlighting-code-blocks/#fenced-code-blocks) with `bash` as the type instead?